### PR TITLE
fix: [OSM-1016] Fixing broken support for `netstandardN.N` for `v2` parser

### PR DESCRIPTION
**Pull request by dotkas**
Doing a `dotnet restore` for a project targeting just `netstandard2.0` or the like, and not more specialized than that, will not produce runtime assembly versions to be used in the dependency graph generation.

We're not able to get `dotnet` to give us a more precise information about the specific runtime versions being used, and I suspect it is because of what you see in [this matrix](https://dotnet.microsoft.com/en-us/platform/dotnet-standard#versions).

Basically that .NET cannot get more specific because `netstandard` really encompasses the large amount of different frameworks.

This PR 
* Stops the flow from trying to find `runtimepack.*` information in the `.deps` files
* Adds tests related to this
* Also did some small housekeeping of tests (wrongly named folders and such)

### CODE REVIEW

File: test/parsers/parse-core.spec.ts [hunk 1]
```diff
@@ -24,7 +24,10 @@ describe('when parsing .NET core', () => {
 
         const manifestFile = 'obj/project.assets.json';
         const expectedTree = JSON.parse(
-          fs.readFileSync(path.resolve(projectPath, 'expected.json'), 'utf-8'),
+          fs.readFileSync(
```
Comment (dotkas) [inline test/parsers/parse-core.spec.ts:27]: Fixtures were mixing `expected` with `expected_depgraph`, so my consistency trigger kicked in.
